### DISCUSSION TITLE
Recipes for https://anaconda.org/beeware/repo

### DISCUSSION THREAD

**Discussion by FeodorFitsner** [beeware/mobile-forge#44] [Q&A]
Could you please point me to the location for iOS wheel recipes published here: https://anaconda.org/beeware/repo

Thank you!

**Comment by freakboy3742** [comment]
I'm a little confused... *this repository* is where the wheel recipes are published. 

The "fat wheels" (the wheels that refer to `iphonesimulator`, rather than `iphonesimulator_arm64`) were generated using a [prototype version of this project](https://github.com/freakboy3742/chaquopy/tree/iOS-support/server/pypi/packages) that was effectively a fork of Chaquopy; there might be some hints that can be gained from those recipes, but a lot has changed in how Cpython runs on iOS since that repo was in use.

**Reply by FeodorFitsner** [reply]
Got it - those iOS wheels were produced with a fork of Chaquopy. Thanks for clarifying!

**Reply by khpeterson** [reply]
I'm looking for a recipe to build/patch ruamel.yaml.clib - is that coming soon to this repository?  The published builds at  https://anaconda.org/beeware/repo are not compatible with briefcase (just file naming inconsistencies, I believe).

**Reply by freakboy3742** [reply]
The file naming "inconsistency" isn't an inconsistency, it's a different type of wheel. As noted in my original reply, there are older style "fat wheels" (using the `iphonesimulator` tag), and newer style "thin" wheels (using the `iphonesimulator_arm64`, `iphonesimulator_x86-64` and `iphoneos_arm64` tags). The `ruamel-yaml-clib` wheel is in the older "fat" format.

If there is a wheel in "fat" format that you need, you may be able to port it over; [this recipe from the old repository](https://github.com/freakboy3742/chaquopy/tree/iOS-support/server/pypi/packages/ruamel-yaml-clib) as a starting point for building wheels and contributing a ruamel-yaml-clib recipe. My current focus is on getting the changes needed to support binary wheels upstream, rather than porting specific recipes; so if you have an immediate need for a recipe, you'll likely need to work on it yourself.

**Reply by khpeterson** [reply]
Thank you for the quick reply, understood.  I have found a work around for ruamel.yaml.clib in the short term but may need to come back to it and work on a thin version.  

I have a recipe for another package in the works and will make a pull request for that one soon as well.

Thank you for all of your efforts on this project.
